### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by taras-oleksiuk**

### CODE REVIEW

File: src/main/java/core/basesyntax/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String greeting = "";
```
Comment (Resci) [inline src/main/java/core/basesyntax/HelloWorld.java:8]:
```suggestion
    private String greeting;
```

File: src/main/java/core/basesyntax/Model/Cat.java [hunk 2]
```diff
@@ -7,10 +7,7 @@
 
     @Override
     public String toString() {
-        return "Cat{" +
-                "age=" + age +
-                ", name='" + name + '\'' +
-                ", ownerName='" + ownerName + '\'' +
-                '}';
+        return "Cat{ age=" + age + ", name='" + name + '\''
+                + ", ownerName='" + ownerName + '\'' + '}';
```
Comment (Resci) [inline src/main/java/core/basesyntax/Model/Cat.java:11]:
```suggestion
        return "Cat{ "
        		 + "age=" + age 
        		 + ", name='" + name + '\''
                 + ", ownerName='" + ownerName + '\'' + '}';
```

File: src/main/java/core/basesyntax/Model/Dog.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package core.basesyntax.Model;
+package core.basesyntax.model;
```
Comment (Resci) [inline src/main/java/core/basesyntax/Model/Dog.java:1]: Назва пакету має співпадати з назвою папки

**Review by Resci — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/HelloWorld.java (hunk 1 above)
Comment (taras-oleksiuk) [reply src/main/java/core/basesyntax/HelloWorld.java]: This has been fixed.

File: src/main/java/core/basesyntax/Model/Cat.java (hunk 2 above)
Comment (taras-oleksiuk) [reply src/main/java/core/basesyntax/Model/Cat.java]: This has been fixed.

File: src/main/java/core/basesyntax/Model/Dog.java (hunk 3 above)
Comment (taras-oleksiuk) [reply src/main/java/core/basesyntax/Model/Dog.java:1]: I see "core.basesyntax.model." everywhere in IDE. I checked top code in classes: Dog, Cat, HelloWorld. Where & how I can fix it?

Comment (MaliukDaria) [reply src/main/java/core/basesyntax/Model/Dog.java:1]: yes, let's rename the package `Model` to `model`.

**Review by MaliukDaria — CHANGES_REQUESTED** [review]

**Pull request closed without merging**